app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_counter(id)
+    post = Post.find_by(id: id)
+    post.likes_counter += 1
+    post.save
+  end
```

Comment:
- [OPTIONAL] Same issue here as with the previous comment.

```suggestion
  after_save :update_counter

  private

  def update_counter
    user.increment!(:comments_counter)
  end
```